portfolio/src/main/java/com/google/sps/servlets/AuthenticationServlet.java

```diff
@@ -13,27 +16,52 @@
 /** Servlet that handles user authentication, so that users must log in to post comments */
 @WebServlet("/login") 
 public class AuthenticationServlet extends HttpServlet {
-  //TODO: change to actual redirect page. currently just redirects to test logoff
-  private final static String REDIRECT_URL = "/login.html"; 
-  private static final String JSON_CONTENT_TYPE = "application/json;";
+  private final static String DISPLAY_NAME_URL = "/displayname.html";
+  // TODO: change to actual logoff redirection page
+  private final static String REDIRECT_LOGOFF_URL = "/login.html";
```

Comment:
use consistent ordering of "private static final" (i.e. don't use private static final for one variable and then private final static for the next)